src/lab13.py

```diff
@@ -829,6 +832,8 @@ def getAttribute(self, handle, attr):
     def innerHTML_set(self, handle, s):
         doc = HTMLParser(
             "<html><body>" + s + "</body></html>").parse()
+        add_field(doc, "style", lambda x: {})
```

Comment:
I thought we had agreed to go back to putting them on the tab and pass some parameters
to `animate`?

Reply:
I think putting animations on tab and passing parameters to animate were separate points.

Passing parameters was the plan for moving the animation loop. I did initially do that, but it lead to code duplication (for the set_needs_layout / set_needs_composite logic). Eventually I hit on the one-more-frame logic to fix that.

But now that I think about it, I think we can put the fields on Element/Text with an even simpler hack than this `add_field` stuff, which should also be very maintainable.

Reply:
ok I'll wait for that..